plugin.cfg

```diff
@@ -20,7 +20,7 @@
 # log:log
 
 tls:tls
-#reload:reload # enable by reverting #1511
+reload:reload
```

Comment:
Hum.. I am not sure when we want to setup the 'reload' plugin.
this UT are good.

But I though were link to deeper tests and review compatibility with other plugins.

Reply:
I have enabled Reload in order for the tests to pass.